src/App.tsx

```diff
@@ -33,78 +51,20 @@ export const App: React.FC = () => {
       </div>
 
       {/* Move this markup to Pagination */}
```

Comment:
Empty JSX block `{}` found. If this is not intended for future use, it should be removed to keep the code clean.